### ISSUE TITLE
Add kt/json examples to JsonBuilder configuration properties

### ISSUE THREAD

**Issue by carstenhag** [Kotlin/kotlinx.serialization#2162]
**What is your use-case and why do you need this feature?**

I wanted to see what the option `explicitNulls` https://kotlinlang.org/api/kotlinx.serialization/kotlinx-serialization-json/kotlinx.serialization.json/-json-builder/explicit-nulls.html (and other options) changes to the (de)serialization behavior.

While it is described in words, kotlin/JSON examples help a lot to grasp it in "easier terms". 

**Describe the solution you'd like**

Other kotlin api references like the following have code examples/embedded playgrounds:
* https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/any.html

**Comment by sandwwraith** [comment]
I'm not sure it is possible currently to embed runnable examples to arbitrary documentation outside of stdlib. In the meanwhile, you can see a lot of examples in the `docs` folder with guide, e.g. https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/json.md#explicit-nulls

**Comment by qwwdfsad** [comment]
True; still would be nice to have basic documentation right in the quickdoc/KDoc, so folks won't be forced to read the guide
